### PR TITLE
MAGECLOUD-1897: Change .htaccess and .gitkeep in templates

### PR DESCRIPTION
**Pull request by NadiyaS**
https://magento2.atlassian.net/browse/MAGECLOUD-1897

Rename .htaccess and .gitkeep files to README.md with adding appropriate description

### CODE REVIEW

File: m2-hotfixes/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ - patches are applied from the root of Magento project. E.g.
+ 
+    ```
+    diff -Naur a/vendor/package-name/ClassName.php b/vendor/package-name/ClassName.php
+    --- a/vendor/package-name/ClassName.php
+    +++ b/vendor/package-name/ClassName.php
+    ...
+    ```
+    
+To test applying all patches (your custom patches and cloud patches) which is made during the deployment, you can run next command locally:
```
Comment (meker12) [inline m2-hotfixes/README.md]:
```suggestion
You can use the following command to test the patch process–applying all Magento patches and your custom patches  locally before you build and deploy to a Cloud environment.
```

File: m2-hotfixes/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Add custom patches
+
+You can add custom patch files to this directory. Each patch file must use the `.patch` extension, for example the `<patch_name>.patch`.
```
Comment (meker12) [inline m2-hotfixes/README.md]:
```suggestion
You can add custom patch files to this directory. You must use the `.patch` extension in the custom patch file name, for example `<patch_name>.patch`.
```

File: m2-hotfixes/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+There are rules about patches in this directory:
```
Comment (meker12) [inline m2-hotfixes/README.md]: After all Magento patches are applied, custom patches are applied in alphabetical order from the root of the Magento project as shown in the following example:

File: m2-hotfixes/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+ - patch file should have extension `.patch`
```
Comment (meker12) [inline m2-hotfixes/README.md]:
```suggestion
```

File: m2-hotfixes/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+ - patches are applied in alphabetical sequence
```
Comment (meker12) [inline m2-hotfixes/README.md]:
```suggestion
 - patches are applied in alphabetical sequence
```
```suggestion
```

File: m2-hotfixes/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+ - patches are applied from the root of Magento project. E.g.
```
Comment (meker12) [inline m2-hotfixes/README.md]:
```suggestion
```

File: m2-hotfixes/README.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    ```
+    diff -Naur a/vendor/package-name/ClassName.php b/vendor/package-name/ClassName.php
+    --- a/vendor/package-name/ClassName.php
+    +++ b/vendor/package-name/ClassName.php
+    ...
+    ```
+    
+You can use the following command to test the patch process–applying all Magento patches and your custom patches  locally before you build and deploy to a Cloud environment.
+``` 
+./vendor/bin/ece-tools patch
```
Comment (shiftedreality) [inline m2-hotfixes/README.md]: This is deprecated in `2002.1`

Comment (NadiyaS) [reply m2-hotfixes/README.md]: @shiftedreality what will be instead?

Comment (NadiyaS) [reply m2-hotfixes/README.md]: and if we plan to deliver these changes before 2002.1 then better not to mention this not released version

Comment (shiftedreality) [reply m2-hotfixes/README.md]: Direct call to patches `./vendor/bin/ece-patches apply`

Comment (NadiyaS) [reply m2-hotfixes/README.md]: @YPyltiai will we deliver this earlier that minor version of ece-tools? or wait till its release and add command as for 2002.0 as for 2002.1?

File: m2-hotfixes/README.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+After all Magento patches are applied, custom patches are applied in alphabetical order from the root of the Magento project as shown in the following example:
+
+    ```
+    diff -Naur a/vendor/package-name/ClassName.php b/vendor/package-name/ClassName.php
+    --- a/vendor/package-name/ClassName.php
+    +++ b/vendor/package-name/ClassName.php
+    ...
+    ```
+    
+You can use the following command to test the patch process–applying all Magento patches and your custom patches  locally before you build and deploy to a Cloud environment.
```
Comment (bbatsche) [inline m2-hotfixes/README.md]: Suggested change:

```
You can use the following command to apply all Magento patches and your custom patches locally.
```

The other wording is awkward, and honestly I don't think you need any more specific details than that.

Comment (meker12) [inline m2-hotfixes/README.md]:
```suggestion
You can use the following command to apply all Magento patches and your custom patches locally.
```

Comment (NadiyaS) [reply m2-hotfixes/README.md]: done.

**Pull request merged**